### PR TITLE
Removed deprecated methods

### PR DESCRIPTION
**Pull request by turtledreams**

### CODE REVIEW

**Comment by turtledreams** [comment]
[[RN] Removed Deprecated](https://trello.com/c/y4uwp2QY/1150-rn-removed-deprecated)

File: Countly.js [hunk 1]
```diff
@@ -1604,19 +1594,21 @@ Countly.getRemoteConfigValueForKeyP = function (keyName) {
...
-    return promise.then((value) => {
-        if (Platform.OS == 'android') {
-            try {
-                value = JSON.parse(value);
-            } catch (e) {
-                // Countly.logError('getRemoteConfigValueForKeyP', e.message);
-                // noop. value will remain string if not JSON parsable and returned as string
+    return promise
+        .then((value) => {
+            if (Platform.OS == 'android') {
```
Comment (ijunaid) [inline Countly.js:1599]: I think we should use the same approach of platform/OS checks for consistency.
i.e "/android/.exec(Platform.OS)"
There are two other places in this file which are using == to check OS on line 1574 and 1593

**Review by ijunaid — CHANGES_REQUESTED** [review]

File: Countly.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ const ratingWidgetCallbackName = 'ratingWidgetCallback';
 const pushNotificationCallbackName = 'pushNotificationCallback';
 
 Countly.messagingMode = { 'DEVELOPMENT': '1', 'PRODUCTION': '0', 'ADHOC': '2' };
-if (Platform.OS.match('android')) {
+if (/android/.exec(Platform.OS)) {
```
[the remark this answers is not in the document]
Comment (turtledreams) [reply Countly.js:42]: linter changed it :/

[the remark this answers is not in the document]
Comment (turtledreams) [reply Countly.js:42]: I am positive changing it back and such

File: Countly.js (hunk 1 above)
Comment (turtledreams) [reply Countly.js:1599]: great catch

**Review by ArtursKadikis — APPROVED** [review]

**Pull request merged**